avaframe/ana3AIMEC/aimecTools.py

```diff
@@ -114,6 +114,17 @@ def fetchReferenceSimNo(pathDict, cfgSetup):
         pathDict['referenceFile'] = indexRef
         log.info('Reference Simulation is based on %s = %s - closest value found is: %s' %
                  (cfgSetup['varParList'].split('|')[0], cfgSetup['referenceSimValue'], str(colorValues[indexRef])))
+    elif cfgSetup['referenceSimName'] != '':
```

Comment:
Please add a short inline comment about what is going on here

Reply:
added a comment